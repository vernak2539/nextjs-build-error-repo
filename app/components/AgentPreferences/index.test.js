import React from 'react';
import { shallow } from 'enzyme';

import { AgentPreferences } from '.';

describe('AgentPreferences', () => {
  const props = {
    countries: {
      au: 'Australia',
      de: 'Germany',
      es: 'Spain',
      fr: 'France',
      uk: 'United Kingdom',
    },
    countryCode: 'uk',
    onModalClose: () => {},
    onUpdateConfig: (config) => config,
    languages: {
      de: 'Deutsch',
      en: 'English',
      es: 'Español',
      fr: 'Français',
    },
    preferredLanguage: 'en',
    isOpen: false,
    onClose: () => {},
    onSubmit: () => {},
  };

  it('should render as expected', () => {
    const element = shallow(<AgentPreferences {...props} />);

    expect(element).toMatchSnapshot();
  });
});
