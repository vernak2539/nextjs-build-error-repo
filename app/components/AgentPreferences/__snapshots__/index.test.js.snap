// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AgentPreferences should render as expected 1`] = `
<Modal
  appElement="#__next"
  headerTitle=""
  isOpen={false}
  onClose={[Function]}
  renderFooter={[Function]}
  type="dialog"
>
  <withSpacing(Heading)
    level={1}
    size="small"
    spacing="0 0 large 0"
  >
    settings.settings_title
  </withSpacing(Heading)>
  <InputLabel
    className=""
    isRequired={false}
    isValid={true}
  >
    settings.select_country_title
  </InputLabel>
  <withLayout(withSpacing(Select))
    id="select-agent-country"
    layout="block"
    onChange={[Function]}
    spacing="0 0 large 0"
    value="uk"
  >
    <Option
      className=""
      highlighted={false}
      isButton={false}
      isMulti={false}
      key="au"
      selected={false}
      value="au"
    >
      Australia
    </Option>
    <Option
      className=""
      highlighted={false}
      isButton={false}
      isMulti={false}
      key="de"
      selected={false}
      value="de"
    >
      Germany
    </Option>
    <Option
      className=""
      highlighted={false}
      isButton={false}
      isMulti={false}
      key="es"
      selected={false}
      value="es"
    >
      Spain
    </Option>
    <Option
      className=""
      highlighted={false}
      isButton={false}
      isMulti={false}
      key="fr"
      selected={false}
      value="fr"
    >
      France
    </Option>
    <Option
      className=""
      highlighted={false}
      isButton={false}
      isMulti={false}
      key="uk"
      selected={false}
      value="uk"
    >
      United Kingdom
    </Option>
  </withLayout(withSpacing(Select))>
  <InputLabel
    className=""
    isRequired={false}
    isValid={true}
  >
    settings.select_language_title
  </InputLabel>
  <withLayout(withSpacing(Select))
    id="select-agent-language"
    layout="block"
    onChange={[Function]}
    spacing="0 0 medium 0"
    value="en"
  >
    <Option
      className=""
      highlighted={false}
      isButton={false}
      isMulti={false}
      key="de"
      selected={false}
      value="de"
    >
      Deutsch
    </Option>
    <Option
      className=""
      highlighted={false}
      isButton={false}
      isMulti={false}
      key="en"
      selected={false}
      value="en"
    >
      English
    </Option>
    <Option
      className=""
      highlighted={false}
      isButton={false}
      isMulti={false}
      key="es"
      selected={false}
      value="es"
    >
      Español
    </Option>
    <Option
      className=""
      highlighted={false}
      isButton={false}
      isMulti={false}
      key="fr"
      selected={false}
      value="fr"
    >
      Français
    </Option>
  </withLayout(withSpacing(Select))>
</Modal>
`;
